userstorage/backend.py

```diff
@@ -92,12 +96,21 @@ def teardown(self):
         Subclass should override if needed.
         """
 
+    def missing_handler(self, msg):
+        """
+        Should be called when entering the context without having created
+        the associated storage.
+
+        Subclass can override if needed.
+        """
+        raise Missing(msg)
```

Comment:
Making this a global function will allow users to change the missing handler easliy:

    import userstorage
    import pytest

    userstorage.missing_handler = lambda msg: pytest.xfail(msg)